src/System.Net.WebHeaderCollection/src/System/Net/HeaderInfo.cs

```diff
@@ -3,13 +3,11 @@
 
 namespace System.Net
 {
-    internal delegate string[] HeaderParser(string value);
-
     internal class HeaderInfo
     {
         internal readonly bool IsRequestRestricted;
         internal readonly bool IsResponseRestricted;
-        internal readonly HeaderParser Parser;
+        internal readonly Func<string, string[]> Parser;
```

Comment:
These fields should all be named like "_parser", "_isRequestRestricted", etc., right?  Or have we exempted them from the guidelines because we're thinking of them more as properties or something?